scripts/ping_forever.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+#
+# The script is expected to run on admin server, takes IP as only argument.
+#
+# This is used for pinging given IP indefinitely (more specifically,
+# until interrupted by creating /var/lib/crowbar/stop_pinging.$IP file).
+#
+# Originally intended for testing the instances accessibility during the Cloud upgrade.
+
+trap '' 1
+
+fip=$1
+LOGFILE=/var/log/ping_instance.$fip.out
+failed=0
+date -Iseconds > $LOGFILE
+echo "Starting the ping of $fip" >> $LOGFILE
+
+while [ 1 = 1 ]; do
+    if [ -e /var/lib/crowbar/stop_pinging.$fip ]; then
+        echo "Stopping the ping of $fip" >> $LOGFILE
+        exit 0
+    fi
+    if ping $fip -W 1 -w 1 > /dev/null ; then
+        if [ "$failed" -gt "0" ]; then
+            date -Iseconds >> $LOGFILE
+            echo "$fip not available for: $failed" >> $LOGFILE
```

Comment:
The $failed counter does not have useful information, as it may be seconds but may also be fractions of that (even mixed). So just rely on the date output.